ci/gpu/build.sh

```diff
@@ -51,7 +51,8 @@ gpuci_logger "Activate conda env"
 conda activate rapids
 gpuci_mamba_retry install "cudatoolkit=${CUDA_REL}" \
               "cudf=${RAPIDS_VERSION}" "dask-cudf=${RAPIDS_VERSION}" \
-              "rapids-build-env=${RAPIDS_VERSION}"
+              "rapids-build-env=${RAPIDS_VERSION}" \
+	      "versioneer>=0.24"
```

Comment:
No because the `ucx-py` packages are not cached after build jobs for reuse in testing jobs. Instead testing jobs build from source as well. Raised issue ( https://github.com/rapidsai/ucx-py/issues/897 ) about enabling this caching.